src/App.tsx

```diff
@@ -20,12 +43,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
```

Comment:
it's better to create a separate function instead of using an anonymous function in such cases - just FYI